provisioner/terraform/provision_test.go

```diff
@@ -703,6 +703,45 @@ func TestProvision(t *testing.T) {
 				}},
 			},
 		},
+		{
+			Name: "workspace-owner-login-type",
+			Files: map[string]string{
+				"main.tf": `terraform {
+					required_providers {
+					  coder = {
+						source  = "coder/coder"
+						version = "1.0.5"
```

Comment:
note: to be released

Reply:
@matifali what version constraint should this specify?

Reply:
Its a new attribute we are adding to an existing resource so as per t[erraform best practices](https://developer.hashicorp.com/terraform/plugin/best-practices/versioning#example-minor-number-increments) we should bumpt a minor version.
I would call it 1.1.0